src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -266,7 +266,7 @@ ul.alt li.warning {
     margin-bottom: 1px;
 }
 ul.sf-call-stack li {
-    text-size: small;
+    font-size: small;
```

Comment:
Not sure about that. intention was clear, but in browsers it may looks a bit different.
For sure it is worth to fix prop name / remove invalid property - does the profiler looks nice with that fix?